versioned_docs/version-v2.18.x/user-guide/configuring-overview copy.md

```diff
@@ -0,0 +1,66 @@
+# Configuring Overview
+
+Review this article for an overview of the procedures that must be performed to configure Zowe z/OS components and the z/OS system. More details about the individual procedures are provided in the articles in this section. 
+
+:::info Required roles: system programmer, security administrator
+:::
+
+Configuring Zowe z/OS components consists of the following four main steps:
+
+1. Configure Zowe runtime
+2. Configure the z/OS system for Zowe
+3. Assign security permissions
+4. Configure the Zowe cross memory server (ZIS)
+
+:::note
+Successful completion of steps 2, 3, and 4 may require elevated security permissions. We recommend you consult with your security administrator to assist with performing these steps. 
+:::
+
+## Configuring Zowe runtime
+
+To cofigure Zowe runtime, choose from the following options:
+
+* **Option 1: Configure Zowe manually using the `zwe init` command group**  
+To run the `zwe init` command, it is necessary to create a Zowe configuration file. For more information about this file, see the [Runtime directory](./installandconfig.md#runtime-directory) which details all of the started tasks in the article _Preparing for installation_.
+
+Once your configuration file is prepared, see [Configuring Zowe with zwe init](./initialize-zos-system.md), for more information about using the `zwe init` command group.
+
+* **Option 2: Configure Zowe via JCL**
+You can configure Zowe by directly customizing JCLs. The Zowe Runtime Dataset `SZWESAMP` contains JCL samples that have templates referencing `zowe.yaml` parameters. These samples should not be submitted without modification.
+
+For more information, see [Configuring Zowe via JCL](./configuring-zowe-via-jcl.md)
+
+* **Option 3: Configure Zowe with z/OSMF workflows**  
```

Comment:
I believe we would want to insert this ection in the real `configuring-overview.md`